src/asmcnc/skavaUI/screen_homing_active.py

```diff
@@ -108,7 +108,10 @@ def on_pre_enter(self):
 
     def on_enter(self):
         if sys.platform == "win32" or sys.platform == "darwin":
-            return
+            if self.m.s.is_connected:
```

Comment:
Do you need to check is_connected after checking platform? I assume if not connected, this function should do nothing. So it could be..: 

if not self.m.s.is_connected:
    return

....

Reply:
But if you want to avoid additional testing, ignore me :)